### PR TITLE
Adding kching.app v1.0.0

### PR DESCRIPTION
**Pull request by skarnecki**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

Adding github.com/skarnecki/kching to casks

### CODE REVIEW

File: Casks/kching.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'kching' do
+  version '1.0.0'
+  sha256 '136f544a784d4bc6a8a01e5b2726fdde575ffd43c9cd24cd13e5aa1ea9fe9940'
+
+  url 'https://github.com/skarnecki/kching/releases/download/1.0/kching-1.0.0.tar.gz'
```
Comment (miccal) [inline Casks/kching.rb]: Please change this to "url 'https://github.com/skarnecki/kching/releases/download/#{version.major_minor}/kching-#{version}.tar.gz"`.

Comment (miccal) [inline Casks/kching.rb]: Please add an [appcast](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

File: Casks/kching.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  name 'kching'
+  homepage 'https://github.com/skarnecki/kching'
+
+  app ''
```
Comment (miccal) [inline Casks/kching.rb]: Please update the [`artifact` stanza](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/all_stanzas.md#at-least-one-artifact-stanza-is-also-required).

**Comment by skarnecki** [comment]
@miccal Thanks for comments - all of them fixed. Tested and linted

**Review by miccal — APPROVED** [review]

**Pull request merged**